### DISCUSSION TITLE
Command to change in the doc to create the credential-store?

### DISCUSSION THREAD

**Discussion by talcher** [Keyfactor/ejbca-ce#330] [Ideas]
Hello,

I am working on the installation of EJBCA v8.0CE on a Wildfly v26.1.3 server.

When I create the credential store with the documentation-suggested command
~~~
/opt/wildfly/bin/jboss-cli.sh --connect '/subsystem=elytron/credential-store=defaultCS:add(location=keystore/credentials, relative-to=jboss.server.config.dir, credential-reference={clear-text="{EXT}/usr/bin/wildfly_pass", type="COMMAND"}, create=true)'
~~~

I get this log:
> INFO  [org.jboss.as.controller.management-deprecated] (management-handler-thread - 2) WFLYCTL0028: Attribute 'location' in the resource at address '/subsystem=elytron/credential-store=defaultCS' is deprecated, and may be removed in a future version. See the attribute description in the output of the read-resource-description operation to learn more about the deprecation.

I guess this new command would do the job:
~~~
/opt/wildfly/bin/jboss-cli.sh --connect '/subsystem=elytron/credential-store=defaultCS:add(path=keystore/credentials, relative-to=jboss.server.config.dir, credential-reference={clear-text="{EXT}/usr/bin/wildfly_pass", type="COMMAND"}, create=true)'
~~~

Let me know what you think about it!
Good bye,

talcher

**Comment by primetomas** [comment]
So location has changed to path it looks like right? If that's the case, thank you. We should update the documentation.,

**Reply by talcher** [reply]
Hello Tomas,
that is it, indeed.
I wanted to write "path" in bold, but that did not work. I edited my comment a few seconds ago, to suppress the misleading **.

**Reply by primetomas** [reply]
I have updated the documentation for the next release.

Thanks a lot for the report!
